### DISCUSSION TITLE
Simultaneous connection between the extension and the web interface with macOS 13 (Ventura)

### DISCUSSION THREAD

**Discussion by ghost** [dani-garcia/vaultwarden#3665] [Q&A]
### Subject of the issue

- live sync for  emergency contacts **does not work**.

When a user using "macOS 13 (Ventura)" is connected to the extension, it works fine, but if they also want to connect to the web interface, they can't find any password, it's completely empty. Even the organisation safe has disappeared. However, on the admin side, the user is still present and in the extension he has access to all his passwords, and even those of his organisation.

### Deployment environment

- vaultwarden version: 1.28.1-alpine
- Install method: docker
- Clients used: all (tested: Browser Chrome)
- Reverse proxy and version: nginx proxy manager-v2.9.19
- Other relevant details: websocket works

### Steps to reproduce
- Use macos ventura
- Connect to the chrome extension
- Connect to the chrome browser web interface
- Bad Enjoy !

### Expected behaviour
Users should be able to connect simultaneously to all possible interfaces (extension, Desktop + web interface) without any problems.

### Actual behaviour
- Users cannot connect simultaneously to all interfaces.

BR Shydoo

**Comment by BlackDex** [comment]
There is no live sync for emergency contacts.
I don't know what you mean with _connect simultaneously to all interfaces_, but i can connect with multiple clients to the same environment at the exact same time, and if i update a cipher it gets updated on all connected clients.

Now i did not used MacOS, but i did tried chrome, and it seems to work just fine for me.

I would suggest to check the developer console of the browser, and maybe also debug the extension to see if there are any error messages there.

It might be wrong headers because of nginx settings or extensions like ModSecurity or WAF.

**Reply by ghost** [reply]
The user logs on to the Chrome extension and has his login details, but when he logs on to the web interface, there's absolutely nothing left in his safe and the organisation has disappeared. But in the extension, everything is OK, including the organisation to which he is attached and his passwords are still there.

I'm on win 11 pro and it works perfectly well with the same tools, but the user is on macOS (Ventura) and it doesn't work.
